src/app/workflow/register-workflow-modal/register-github-app-modal/register-github-app-modal.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="mb-3" *ngIf="Dockstore.FEATURES.enableNotebooks && entryType === EntryType.Notebook">
+  <app-preview-warning></app-preview-warning>
+</div>
+<p>
+  Navigate to GitHub to install our GitHub app on your repositories/organizations. See our
+  <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps/github-apps.html" target="_blank" rel="noopener noreferrer">
+    documentation <mat-icon>open_in_new</mat-icon>
+  </a>
+  for how to use GitHub Apps to automatically sync your {{ entryType }} with GitHub. A .dockstore.yml file is required.
+</p>
+<div fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="2rem" class="mb-2">
+  <a
+    mat-raised-button
+    type="button"
+    class="accent-1-dark-btn mat-elevation-z"
+    [href]="gitHubAppInstallationLink$ | async"
+    [disabled]="usernameChangeRequired$ | async"
+    matTooltip="Manage Dockstore installations on GitHub"
+    ><mat-icon>add</mat-icon> Manage Dockstore installations on GitHub</a
```

Comment:
I think this link might have a leading space (I remember seeing something similar on QA)

Reply:
Yeah, that's the right one. When you hover over the link, you can see an underline for the space, which IMO looks awkward:
![image](https://user-images.githubusercontent.com/25287123/233200360-d7e22c5e-ff3c-4007-87ed-fa3884023ede.png)


Does it make a huge difference without the space? If not, I would remove it. If it does, consider adding a right margin to the icon instead of the space. 

Admittedly, this is a very minor thing so your call whether or not you think it's worth doing :slightly_smiling_face:

Reply:
Ah I see, in that case I can add the `no-underline` style which gets rid of the underline on hover which I think looks better for links styled as buttons :)